drivers/sensor/lis2dh/lis2dh.h

```diff
@@ -193,12 +193,10 @@ struct lis2dh_config {
 	int (*bus_init)(const struct device *dev);
 	const union lis2dh_bus_cfg bus_cfg;
 #ifdef CONFIG_LIS2DH_TRIGGER
-	const char *irq1_dev_name;
-	gpio_pin_t irq1_pin;
-	gpio_dt_flags_t irq1_flags;
-	const char *irq2_dev_name;
-	gpio_pin_t irq2_pin;
-	gpio_dt_flags_t irq2_flags;
+	const struct gpio_dt_spec gpio_drdy;
+	const struct gpio_dt_spec gpio_int;
+	bool trig_drdy_en;
+	bool trig_int_en;
```

Comment:
I think these fields are not needed and can be removed.

It seems like `cfg->trig_drdy_en` is the same thing as `cfg->gpio_drdy.port != NULL`, and similarly for `trig_int_en`.

Reply:
Yes, I could test (cfg->gpio_drdy.port != NULL). It is the same thing in fact.
OK, I'll remove the flags.